packages/forma-36-website/src/pages/foundation/layout/workbench/main-content/index.mdx

```diff
@@ -0,0 +1,32 @@
+import skeleton from './maincontent-skeleton.jpg';
+
+## Main Content
+
+The main content area provides the main functionality of the current view.
+
+<img src={skeleton} width="100%" />
+
+### Content and interaction Guidelines
+
+Depending on the displayed information one of the 3 width configurations should be applied:
+
+- Views that just consist of forms or text should have a maximum width of 786px for enhanced readability.
+- Static table views which contents are not configurable should default to a maximum width of 1240px.
+- Rich data views like dashboards or configurable tables should use the entire available width.
+
+### Width utility classes
+
+| Classname                   | CSS               |
+| :-------------------------- | :---------------- |
+| .f36-content-width--default | max-width: 1240px |
+| .f36-content-width--text    | max-width: 786px  |
```

Comment:
As a nod to the old XGA resolution of 1024 x 768px, this should be 768px instead of 786px :)